### DISCUSSION TITLE
repalce matrix in ArchR

### DISCUSSION THREAD

**Discussion by qi825** [GreenleafLab/ArchR#1629] [General]
Is there any possibility that l could replace the PeakMatrix in ArchR with the matrix generated some other place?
Thank you for your help!

**Comment by rcorces** [comment]
No, this is not currently possible. But you can replace your peak set with `addPeakSet()` and then make a new PeakMatrix from that using `addPeakMatrix()`

**Comment by qi825** [comment]
That's a pity. Thank you anyway!

**Comment by DruncBread** [comment]
Does it support this feature now?
